packages/client/src/products/types/productGrouping.types.ts

```diff
@@ -0,0 +1,31 @@
+export type DigitalAsset = {
+  mediaType: string;
+  displayOrder: number;
+  size: string;
+  url: string;
+  type: number;
+};
+
+export type ProductGroupingEntry = {
+  digitalAssets: DigitalAsset[];
+  variationProperties: Array<{
+    type: 'Volume' | 'COLOR';
```

Comment:
I know this is probably what is returned by the api, but do you think this will be fixed and should we leave a comment here?

Reply:
Yup, this is how the API returns the types and I'm not confident it will change in the near future :/ so I added a comment saying that values come straight from the API.